src/Components/Admins/Error/error.module.css

```
@@ -0,0 +1,7 @@
+.error{
+  color:red;
+  display: flex;
+  justify-content: start;
+  margin:0;
+  width:80%;
+}
```

Keep consistency with spaces between CSS property names and values.

```suggestion
.error{
  color: red;
  display: flex;
  justify-content: start;
  margin: 0;
  width: 80%;
}
```